darwin/exporter/formats/darwin_1_0.py

```diff
@@ -1,22 +1,23 @@
 from pathlib import Path
-from typing import Any, Dict, Iterable, List, Union
+from typing import Any, Dict, Iterable, Union
 
 import orjson as json
 
 import darwin.datatypes as dt
-from darwin.exporter.formats.numpy_encoder import NumpyEncoder
 
 
 def export(annotation_files: Iterable[dt.AnnotationFile], output_dir: Path) -> None:
     for id, annotation_file in enumerate(annotation_files):
         _export_file(annotation_file, id, output_dir)
 
 
-def _export_file(annotation_file: dt.AnnotationFile, id: int, output_dir: Path):
+def _export_file(annotation_file: dt.AnnotationFile, id: int, output_dir: Path) -> None:
     output: Dict[str, Any] = _build_json(annotation_file)
     output_file_path: Path = (output_dir / annotation_file.filename).with_suffix(".json")
     with open(output_file_path, "w") as f:
-        op = json.dumps(output, option=json.OPT_INDENT_2 | json.OPT_SERIALIZE_NUMPY).decode("utf-8")
+        op = json.dumps(output, option=json.OPT_INDENT_2 | json.OPT_SERIALIZE_NUMPY | json.OPT_NON_STR_KEYS).decode(
```

Comment:
@Fl4m3Ph03n1x Do you have example of such annotation that has non string keys? We never do that, so allowing non string keys feels like a hack, rather then solution. Maybe I'm wrong here, that's why I want to see an example. 
I don't want to block this PR if it unblock clients, but I want us to be aware of tech debt that we're creating (cc @owencjones ).

Reply:
Sorry @rslota didn't see this: have sent you an example